src/_imagingft.c

```diff
@@ -1639,6 +1648,7 @@ PyInit__imagingft(void) {
     }
 
 #ifdef Py_GIL_DISABLED
+    ft_library_mutex = (PyMutex){0};
```

Comment:
That's because assignment without a cast is not possible when it's not initialization.

We can actually get rid of assigning to the mutex altogether by declaring the `PyMutex` as static, which is the recommended way and what needs to be done here. Sorry I missed this and thanks for pointing it out.

Reply:
Just for more information, could you link to that recommendation? I'm not seeing anything explicit at https://docs.python.org/3/c-api/init.html#synchronization-primitives

Reply:
The recommendation is in a header comment in the CPython code [here](https://github.com/python/cpython/blob/dc76a4ad3c46af3fb70361be10b8a791cd126931/Include/cpython/lock.h#L15).